firmware/parse.c

```diff
@@ -142,6 +143,7 @@ void parse_ascii(unsigned char * buf, parse_result_t * res)
 
     if(state == REJECT)
     {
+        com_print(buf);
```

Comment:
This is a functional change and may be out of scope for this PR. In any case, does this break the exiting protocol? It means you'll get garbage back. I think the current scheme expects valid packets only back